deploy-all.yml

```diff
@@ -73,6 +73,7 @@
   roles:
     - common-server
     - forward-server-mail
+    - geerlingguy.nginx
```

Comment:
I usually did not bother to use any nginx role, since it's so simple to make do without.  Installing nginx is a matter of

    - apt:
        name: nginx

After that, you just need to copy a config file and enable it.  If the role works for you, that's fine too, but if it's difficult to make it work I suggest falling back to the manual approach.

Reply:
Since you are relying on this role in {{ansible-load-balancer}}, you should add it to the dependencies of that role (in {{meta/main.yml}}, instead of adding it here.  (That is, if you decide that the role is useful.)

Reply:
OK.

Reply:
I'm taking the approach you recommended of installing Nginx using just the apt module. If that works out I will just close this PR>